aws/table_aws_account.go

```diff
@@ -28,6 +28,12 @@ func tableAwsAccount(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_JSON,
 				Transform:   transform.FromField("Aliases"),
 			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the Account.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromP(accountDataToAkas, "ARN"),
```

Comment:
Is there any reason to use `FromP`?
You can update `accountDataToAkas` to return the `arn`, and use `EnsureStringArray` transform to form the `akas`